contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -576,6 +569,28 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
         return _checkIsAllowedRelayer(delivererAddress, allowedRelayers);
     }
 
+    /**
+     * @dev Returns the number of outstanding receipts that have been received from the given chain ID.
+     */
+    function getOutstandingReceiptQueueSize(bytes32 chainID)
```

Comment:
I think we should add these to the `ITeleporterMessenger` interface as well so that other contracts that want to use them only need to import the interface and not the entire implementation.